Dockerfile.ci

```diff
@@ -123,7 +121,6 @@ ARG RUNTIME_APT_DEPS="\
       krb5-user \
       ldap-utils \
       less \
-      libpython2.7-stdlib \
```

Comment:
@potiuk  do we still need to keep this?

Reply:
I think a better question for this is: should we still enable user to run python2 virtualenv with the `PythonVirtualenvOperator`

Reply:
I checked AIP 3 and found it was already discussed:
https://issues.apache.org/jira/browse/AIRFLOW-4828
 https://github.com/apache/airflow/pull/5449

Reply:
Hi @eladkal, thanks for pointing me to the PR. Let me explain my reason a bit. I don't mean to remove the `python_version` param, and I think we should keep them to allow the user to run different versions of python. The question that I want to discuss is since we drop support for python2, should we still keep the python2 related tests and dependencies in our codebase and CI? IMO, similar to @kurtqq 's opinion in that PR, we should remove python2 related tests and dependencies  but if anything goes wrong, that's user's problem

Reply:
We need to support this case as this is the recommended migration approach for some users who have code that requires libraries only supported by Python 2.7.  Airflow is a task orchestration tool and we dropped Python 2.7 support in the core, but we should still support task orchestrations that use Python 2.7. I think we can rethink this issue when we drop support for Airflow 1.10, because then we can assume that all users have migrated to Airflow 2.0 and all users use Python 3+ as the main version.

Reply:
I think we should keep definitely keep it. Otherwise PythonVirtualenv operator will not work with ptyhon2.7. This is one of the ways we still tell our users how to run their python 2.7 code even if airflow does not support 2.7 as such. And I think this is no harm to keep it.

Reply:
Makes sense! Let's keep it